api/getting-started.md

```diff
@@ -0,0 +1,444 @@
+---
+title: Getting started with the Docker Remote API
+description:
+---
+
+# Getting started with the Docker Remote API
+
+To try out the Docker Remote API in development, [you first need to install Docker](https://docs.docker.com/engine/installation/).
+
+Next, you need to install an SDK for the language you are using. There are official ones available for Python and Go, and a number of community maintained libraries for other languages. [Head to the SDKs page to find and install them.](sdks.md)
+
+## Running a container
+
+The most basic thing you can do with Docker is running a container. On the command line, you would use the `docker run` command, but this is just as easy to do from your own apps too.
+
+This is the equivalent of doing `docker run alpine echo hello world`:
```

Comment:
What if we had 4 tabs instead, the default one being CLI? That would probably make it easier to scan with the eyes and find the example you're looking for.

```
 [CLI][Curl][Python][Go]-------------------
 |
 |   docker run alpine echo hello world
 |
 +-----------------------------------------
```